packages/core/src/index.ts

```diff
@@ -2,14 +2,20 @@ export { buildModule } from "./buildModule";
 export * from "./errors";
 export * from "./initialization";
 export { defineModule } from "./new-api/define-module";
+export { Deployer } from "./new-api/deployer";
```

Comment:
As mentioned in #254 by mistake, many of the newly exported things here should be private.

It's probably time to clean up the repo, as it's getting tricky to understand what should or should not be exported and why.

Reply:
I marked the new things that I believe should be private. I think it would be more clear if they were defined in `internal/`

Reply:
I am going to push on these in this PR. I have added this issue to track:

https://github.com/NomicFoundation/ignition/issues/273